lib/logger.dart

```diff
@@ -42,7 +42,7 @@ class BreezLogger {
     Config.instance().then((config) {
       var appDir = Directory(config.sdkConfig.workingDir);
       _pruneLogs(appDir);
-      final file = File("${_logDir(appDir)}/${DateTime.now().millisecondsSinceEpoch}.app.log");
+      final file = File("${_logDir(appDir)}/${_formatTime(DateTime.now())}.log");
```

Comment:
keep the `.app.` part

Reply:
Ok; but for curiosity, why?

Reply:
It's an identifier we've added to distinguish app logs from extension logs instead of using a separate folder for them.